app/krs.js

```diff
@@ -116,15 +151,18 @@ exports.provisionKey = co(function *(req) {
     }
   }
 
-  let masterKey = yield MasterKey.findOne({ customerId, coin });
+  let masterKey;
 
-  if (!masterKey) {
-    masterKey = yield provisionMasterKey(coin, customerId);
+  if ([ 'xlm', 'txlm' ].includes(coin)) {
```

Comment:
I think we should add coin => recovery type to the config so we can just look it up

Reply:
Similar to above: this isn't really re-usable for other potential future coins because the serialization of keys is different between coins. We would never have any changes to make to that config file.

Reply:
In that case, why even pass 'type' param around everywhere when you could check on coinName like this? If you anticipate other different types, then I still think config file would be helpful

Reply:
Config file specifying what exactly? Every coin that doesn't use xpubs is going to require code changes. I can't think of any way to parameterize key serialization precisely enough that supporting new coins would be as simple as modifying a config file.

Reply:
OK yeah its not a big deal, I mostly just didn't want to be checking for `type` sometimes and `coin = xlm or txlm` other times